src/components/SliderArrows.js

```diff
@@ -109,14 +109,14 @@ export const AltLeftArrow = props => {
       onMouseMove={props.over}
```

Comment:
I have a feeling we can simplify this file as well, so that it works for both cases without containing 4+ components. 
We could start by passing styles in and using Object.assign.